pkg/oc/admin/diagnostics/cluster.go

```diff
@@ -57,31 +63,55 @@ func (o DiagnosticsOptions) buildClusterDiagnostics(rawConfig *clientcmdapi.Conf
 	if err != nil {
 		return nil, false, err
 	}
+	imageClient, err := imageclient.NewForConfig(config)
```

Comment:
@deads2k we have something similar for describer and now here... make me think if we don't want to use the factory here and in describer to avoid this.